services/worker-manager/src/providers/aws.js

```diff
@@ -383,6 +383,12 @@ export class AwsProvider extends Provider {
   }
 
   async removeWorker({ worker, reason }) {
+    await worker.update(this.db, worker => {
```

Comment:
I'd probably rename this inner variable to avoid shadowing and guessing which value is being updated

Reply:
Good call, thanks. It was a copy/pasta, so I updated the original as well 👍🏻